src/Marten.Testing/Events/aggregateto_linq_operator_tests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Threading.Tasks;
+using Marten.Events;
+using Marten.Testing.Events.Projections;
+using Marten.Testing.Harness;
+using Xunit;
+
+namespace Marten.Testing.Events
+{
+    public class aggregateTo_linq_operator_tests: DestructiveIntegrationContext
+    {
+        private readonly MembersJoined _joined1 = new() { Members = new[] { "Rand", "Matrim", "Perrin", "Thom" } };
```

Comment:
I couldn't tell what the style for private fields is supposed to be. Not sure if tests are different than code regarding use of the leading underscore?

Reply:
I wouldn't stress out about that too much, but is there even any reason to have a private member there? Leading underscore in real code of course.

Reply:
The reason for the private fields was that they get used in both the sync and async tests,  But I might be misunderstanding your question.